index.js

```diff
@@ -147,14 +151,17 @@ module.exports = ({utPort, registerErrors}) => class HttpPort extends utPort {
                 requestTimeout: msg.requestTimeout || this.config.requestTimeout || 30000
             };
             if (methodName && this.openApi[methodName]) {
-                Object.assign(reqProps, this.config.raw, this.openApi[methodName](msg));
+                merge(reqProps, {
+                    followRedirect: false,
```

Comment:
Making this default may break existing projects. Usually default is to follow redirects.

Reply:
I am afraid it is the oposit - it was set to false before (talking about standard http request but not ut-openapi). It will be set to true by default if we omit it but I am not sure whether we are going to break something

Reply:
If you are talking about the openapi in particular, yes it was true. So do I make it `true` by default for ut-openapi and `false` by default for all the other cases like it used to be? It doesn't sound right the 2 cases to behave differently by default.

Reply:
Sorry, I did not see it was false before.